optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/SolverBenchmarkConfig.java

```diff
@@ -88,6 +100,10 @@ protected void validate() {
             throw new IllegalStateException("The solverBenchmark name (" + name
                     + ") is invalid because it starts or ends with whitespace.");
         }
+        if (subSingleCount == null || subSingleCount < 1) {
```

Comment:
If it's null it's invalid? If it's null, shouldn't it be valid (and default to 1 basically)?